utility/src/main/java/org/oppia/android/util/parser/html/ListItemLeadingMarginSpan.kt

```diff
@@ -39,14 +38,13 @@ sealed class ListItemLeadingMarginSpan : LeadingMarginSpan {
   ) : ListItemLeadingMarginSpan() {
     private val resources = context.resources
     private val bulletRadius = resources.getDimensionPixelSize(R.dimen.bullet_radius)
-    private val spacingBeforeText = resources.getDimensionPixelSize(R.dimen.spacing_before_text)
-    private val spacingBeforeBullet = resources.getDimensionPixelSize(R.dimen.spacing_before_bullet)
 
     private val bulletDiameter by lazy { bulletRadius * 2 }
+    private val baseMargin = (16f * context.resources.displayMetrics.density).toInt()
```

Comment:
Here and below, why are we using a hardcoded value of 16f here?

Reply:
Apologies for the oversight. I’ve replaced the hardcoded value to improve maintainability.

For **Unordered List (UL)**:
``` 
val baseMargin = context.resources.getDimensionPixelSize(R.dimen.spacing_before_bullet)
```
---
For **Ordered List (OL)**:
``` 
val baseMargin = context.resources.getDimensionPixelSize(R.dimen.spacing_before_number_prefix)
```